### ISSUE TITLE
New data sources, EMAPA & MP

### ISSUE THREAD

**Issue by kevinschaper** [TranslatorSRI/Babel#733]
### What are you requesting?

- [x] Identifier system
- [ ] Mapping source

### Priority

High

### Data source name

Mouse Phenotype Ontology, Mouse Developmental Anatomy Ontology

### Data source URL

https://informatics.jax.org/vocab/mp_ontology/ https://obofoundry.org/ontology/emapa.html 

### Biolink type of these identifiers

biolink:PhenotypicFeature, biolink:AnatomicalEntity

### Value of including this source

For translator's alliance ingest (gene to phenotype, gene expression), we're getting normalization misses on 3889 EMAPA terms and 11,105 MP terms. I think it's likely that these terms will all be clique leaders or unique for the concepts they represent, so hopefully that makes it an easier one to load.  (I'm assuming that you won't normalize to uberon or upheno, and will just keep the species specific terms.)

### Potential conflicts with existing sources

Shouldn't conflict with any existing sources

**Comment by gaurav** [comment]
> (I'm assuming that you won't normalize to uberon or upheno, and will just keep the species specific terms.)

@kevinschaper Wouldn't it make more sense to combine these cliques, using e.g. the cross-references at https://monarchinitiative.org/UBERON:0012313 to combine [UBERON:0012313](http://purl.obolibrary.org/obo/UBERON_0012313) with [EMAPA:16389](http://purl.obolibrary.org/obo/EMAPA_16389)? One reason for doing that is because otherwise you'll end up with both terms if you try searching NameRes for "1st arch maxillary ectoderm". However, when I last tried adding MP (https://github.com/NCATSTranslator/Babel/pull/300), I wasn't able to find comprehensive cross-references: I ended up with 13,335 MP terms that didn't combine with anything else. But if you think that's okay, that would certainly simplify adding these identifiers!

We could also add "(mouse)" to all the EMAPA/MP terms to disambiguate them.

**Comment by cbizon** [comment]
ORION/MGI survey note: we are planning to ingest MGI `MP_EMAPA.rpt` as MP-to-mouse-anatomy bridge data, but current NodeNorm coverage is partial. In the current MGI bridge file we saw 1,645 unique `EMAPA` IDs; live NodeNorm normalized 391 of them, all canonicalizing to `UBERON`, and returned null for 1,254. Examples returning null include `EMAPA:35112` (adipose tissue), `EMAPA:19209` (brown fat), and `EMAPA:16991` (outer ear). Until EMAPA lands in Babel, we will need non-strict normalization in ORION for this ingest so source-local EMAPA IDs are retained rather than dropped.
